validator/engine/validator.js

```diff
@@ -5753,6 +5753,10 @@ amp.validator.ValidationResult.prototype.outputToTerminal = function(
   const {status} = this;
   if (status === amp.validator.ValidationResult.Status.PASS) {
     terminal.info('AMP validation successful.');
+    terminal.info('Important: Valid AMP pages should also be tested for proper '
+        + 'CORS handling when served from the amp cache. For details please '
```

Comment:
`the amp cache` probably should be `an AMP cache`

Reply:
+1

Reply:
done

Reply:
I may be splitting hairs but I believe there is a difference between "the AMP cache" and "an AMP cache". To me, by using "the" implies it means a specific AMP cache, e.g. the Google AMP Cache. If so then we may as well state the "Google AMP Cache". But I believe the original issue applies to any AMP cache, which leads me to a desire to say "an AMP cache".

@CrystalOnScript 	thoughts?

Reply:
no good point, an AMP cache is definitely the right wording.